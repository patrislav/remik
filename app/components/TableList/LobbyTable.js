import React, {PropTypes} from 'react'
import styles from './LobbyTable.css'

const propTypes = {
  table: PropTypes.shape({
    id: PropTypes.string.isRequired
  }).isRequired
}

const LobbyTable = ({ table, ...props }) => (
  <div className={styles.wrapper} {...props}>
    Table #{table.id}
  </div>
)

LobbyTable.propTypes = propTypes

export default LobbyTable
