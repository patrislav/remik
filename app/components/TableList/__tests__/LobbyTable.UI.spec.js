import React from 'react'
import {shallow} from 'enzyme'
import {shallowToJson} from 'enzyme-to-json'

import LobbyTable from '../LobbyTable'

describe('<LobbyTable />', () => {
  it('renders correct UI', () => {
    const table = {
      id: 'ID'
    }
    const wrapper = shallow(<LobbyTable table={table} />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
