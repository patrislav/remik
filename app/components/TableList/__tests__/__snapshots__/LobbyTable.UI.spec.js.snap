exports[`<LobbyTable /> renders correct UI 1`] = `
<div
  className="wrapper">
  Table #
  ID
</div>
`;
